app/assets/stylesheets/active_admin/print.scss

```diff
@@ -228,61 +153,44 @@ table {
   }
 }
 
-// Sidebars
-#filters_sidebar_section {
-  display: none;
-}
-
 // Forms
-form {
-  fieldset {
-    border-top: thick solid #ccc;
-    padding-top: 2em;
-    margin-bottom: 2em;
+fieldset {
+  border: none;
+  margin: 0 0 2em 0;
+  padding: 2em 0 0 0;
 
-    &:last-child {
-      border-bottom: none;
-    }
+  fieldset {
+    border: 1px solid black;
+    padding: 1em;
   }
+}
 
-  .buttons, abbr {
-    display: none;
-  }
-  ol {
+form {
+  ol, li {
     list-style-type: none;
-    padding: 0;
     margin: 0;
+    padding: 0;
+  }
 
-    li{
-      border-top: thin solid black;
-      margin: 0;
-      padding: 1em 0;
-      overflow: hidden;
-
-      &.password, &.hidden {
-        display: none;
-      }
+  li {
+    padding: 1em 0;
 
-      label {
-        font-weight: bold;
-        float: left;
-        width: 20%;
-      }
+    > label,
+    legend label {
+      font-weight: bold;
+    }
 
-      input, textarea, select {
-        background: none;
-        border: 0;
-        font: Arial, Helvetica, sans-serif;
-      }
+    abbr[title] {
+      border-bottom: none;
+      font-weight: normal;
 
-      input[type=file] {
-        display: none;
+      &:before {
+        content: ' (required)';
       }
+    }
 
+    input, textarea, select {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line